### PR TITLE
Fixed broken video capabilities of Connect

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

I did some refactoring and lint fixes in here, but the main issue was that `fullscreen` was undefined. It's a prop that is passed in.

## QA Instructions, Screenshots, Recordings

Before

image

After

image

To test:

* Go to any Connect channel and type in `/call` then press the ENTER key.
* A video session should begin.

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Snippet of a commercial for Kodak VHS and Beta tapes

### CODE REVIEW

File: app/javascript/chat/videoContent.jsx [hunk 1]
```diff
@@ -1,55 +1,69 @@
...
+  const { videoPath, onTriggerVideoContent, fullscreen } = props;
+
+  if (!videoPath) {
+    return null;
+  }
+
+  return (
+    <div
+      data-testid="connect-video"
+      role="presentation"
```
Comment (nickytonline) [inline app/javascript/chat/videoContent.jsx:35]: Fixed some a11y issues like this one.

File: app/javascript/chat/videoContent.jsx [hunk 2]
```diff
@@ -1,55 +1,69 @@
...
+ * Disaplys video in Connect.
+ *
+ * @param {string} props.videoPath The URL of a video
+ * @param {function} props.onTriggerVideoContent Fires when a video is clicked on
+ * @param {boolean} props.fullscreen Whether or not to show the video in fullscreen
+ *
+ * @returns A video component
+ */
+export function VideoContent(props) {
+  const { videoPath, onTriggerVideoContent, fullscreen } = props;
```
Comment (nickytonline) [inline app/javascript/chat/videoContent.jsx:26]: The actual bug was the `fullscreen` prop not being used. It must have been destructured in another PR and somehow that got removed leaving the bare `fullscreen` var as undefined.

File: app/javascript/chat/videoContent.jsx [hunk 3]
```diff
@@ -1,55 +1,69 @@
...
+/**
+ * Disaplys video in Connect.
+ *
+ * @param {string} props.videoPath The URL of a video
+ * @param {function} props.onTriggerVideoContent Fires when a video is clicked on
+ * @param {boolean} props.fullscreen Whether or not to show the video in fullscreen
+ *
+ * @returns A video component
+ */
+export function VideoContent(props) {
```
Comment (nickytonline) [inline app/javascript/chat/videoContent.jsx:25]: This got refactored to a function component.

File: app/javascript/chat/chat.jsx [hunk 4]
```diff
@@ -2,6 +2,8 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
 import ConfigImage from '../../assets/images/overflow-horizontal.svg';
+import { setupPusher } from '../utilities/connect';
```
Comment (nickytonline) [inline app/javascript/chat/chat.jsx:5]: all things in here are lint/prettier in action

File: app/javascript/chat/chat.jsx [hunk 5]
```diff
@@ -34,10 +36,7 @@ import Compose from './compose';
 import Message from './message';
 import ActionMessage from './actionMessage';
 import Content from './content';
-import VideoContent from './videoContent';
-
-import { setupPusher } from '../utilities/connect';
-import debounceAction from '../utilities/debounceAction';
+import { VideoContent } from './videoContent';
```
Comment (nickytonline) [inline app/javascript/chat/chat.jsx:39]: This is no longer a default export.

**Comment by rhymes** [comment]
I'm having troubles on testing this locally as created channels don't appear in my list locally anymore :(

**Pull request merged**